### PR TITLE
AVS pages improvements

### PR DESCRIPTION
**Pull request by Zishan-7**

### CODE REVIEW

File: src/avs/AVSTotalValueOvertime.jsx [hunk 1]
```diff
@@ -42,8 +44,15 @@ const AVSTotalValueOvertime = ({ avsAddress }) => {
       try {
         const avsTotalValueOvertimeData =
           await avsService.getAVSTotalValue(avsAddress);
+
+        const growthPercentage = getGrowthPercentage(
+          avsTotalValueOvertimeData[avsTotalValueOvertimeData.length - 2].tvl,
+          avsTotalValueOvertimeData[avsTotalValueOvertimeData.length - 1].tvl
+        );
```
Comment (vincenthongzy) [inline src/avs/AVSTotalValueOvertime.jsx:51]: this does not look like the correct parameters for growth percentage, we need to get the current slice's first and last values

Comment (vincenthongzy) [reply src/avs/AVSTotalValueOvertime.jsx:51]: e.g 
- for the 7 day slice, get first and last day
- for the 30 day slice ,get first and last day

Comment (vincenthongzy) [reply src/avs/AVSTotalValueOvertime.jsx:51]: image

**Review by vincenthongzy — CHANGES_REQUESTED** [review]

File: src/avs/AVSTotalValueOvertime.jsx (hunk 1 above)
Comment (Zishan-7) [reply src/avs/AVSTotalValueOvertime.jsx:51]: ah, didn't know this

File: src/avs/OperatorsOvertime.jsx [hunk 2]
```diff
@@ -29,7 +29,14 @@ const OperatorsOvertime = ({ avsAddress }) => {
...
     if (!state.operatorsOvertimeData) return null;
-    return getDataByRange();
+    const data = getDataByRange();
+    dispatch({
+      growth: getGrowthPercentage(
+        data[0].operators,
+        data[data.length - 1].operators
+      )
+    });
+    return data;
```
Comment (vincenthongzy) [inline src/avs/OperatorsOvertime.jsx:39]: I don't think this is intended, this should be in a useEffect that changes when filteredData change right?

**Review by vincenthongzy — CHANGES_REQUESTED** [review]

File: src/avs/OperatorsOvertime.jsx (hunk 2 above)
Comment (vincenthongzy) [reply src/avs/OperatorsOvertime.jsx:39]: currently if u change the timeline tabs, the percentage doesn't change

Comment (Zishan-7) [reply src/avs/OperatorsOvertime.jsx:39]: Its working fine at my end 🤔

**Pull request merged**